.travis.yml

```diff
@@ -1,18 +1,24 @@
-matrix:
+services:
+  - docker
+osx_image: xcode10.1
+jobs:
   include:
-    - os: linux
-      dist: trusty
-      sudo: required
+    - stage: Run tests
+      script: ./Scripts/travis/test-Linux.sh
+      os: linux
+      dist: xenial
     - os: osx
-      osx_image: xcode10.1
-branches:
-  except:
-  - gh-pages
-script:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then xcodebuild clean test -scheme RestKit -destination "platform=iOS Simulator,name=iPhone X,OS=12.0"; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get update -y ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then wget https://swift.org/builds/swift-4.1-release/ubuntu1404/swift-4.1-RELEASE/swift-4.1-RELEASE-ubuntu14.04.tar.gz ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then tar xzvf swift-4.1-RELEASE-ubuntu14.04.tar.gz ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then export PATH=swift-4.1-RELEASE-ubuntu14.04/usr/bin:$PATH ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then swift build ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then swift test ; fi
+      script: ./Scripts/travis/test-macOS.sh
```

Comment:
I'm unclear about the step to run the osx tests.  Is it part of the "Run tests" stage?

Reply:
Yes it will run in parallel to the Linux tests